src/components/Summary/SummaryDefineCompany.vue

```diff
@@ -27,6 +27,14 @@
           </div>
         </v-flex>
       </v-layout>
+      <v-layout row v-if="getNameTranslations && getNameTranslations.length" class="mt-3">
```

Comment:
Would optional chaining work here?

Reply:
Optional chaining doesn't seem to work in the template but i will dig into this a bit more and see what the options are!

Reply:
OK, I was unsure. Might be because template code is JS while script is TS.

Not worth spending a lot of time on this.